docs/blog/20230701-Daeraxa-JulyUpdate.md

```diff
@@ -0,0 +1,129 @@
+
+## Subreddit closure/reopening and possible Lemmy community
+
+<img src="./assets/reddit-private.png" height=200><img src="./assets/lemmy-icon.png" height=200>[^1]
+
+You may or may not be aware that we decided to [close our subreddit](https://www.reddit.com/r/pulsaredit/comments/146ja5a/rpulsaredit_is_taking_part_in_the_reddit_blackout/) as part of the protest against the Reddit API changes. The subreddit remained closed after the original 48 hours for various reasons.
+
+We recently decided to [re-open the subreddit](https://www.reddit.com/r/pulsaredit/comments/14ktr3q/rpulsaredit_is_reopening_read_on_for_our/) for a number of reasons, some because we don't want to abandon a portion of our community entirely and partly because of threats from Reddit themselves.
+
+The reason I bring this up here is because it is already possible that some people who were using Reddit for Pulsar news have now left it and won't have seen the above post. This post also contains a poll to get feedback from the affected community to see what our next steps would be. Should we close the subreddit and abandon it? Should we open a Lemmy community instead? Should we do both?
+
+So if you are interested in using either Reddit or Lemmy, please have a look at the above post and vote in the poll.
+
+## Antivirus `es5-ext` issue
+
+<img src="./assets/malware.png" height=200>
+
+Some users have been reporting that various antivirus applications and virus scanning tools have been identifying Pulsar as a virus due to the `es5-ext` package.
+
+This is is nothing to worry about — Pulsar is not a virus, nor do we use dependencies that are.
+
+After some investigation, we discovered that somewhere in our dependency tree (i.e., a package dependency of a package we use) exists this `es5-ext` package. Early last year, this package was updated to display a "message of peace" to Russians (in reference to the ongoing conflict). This message displays to anyone within a Russian time zone, and, whilst the "payload" of a simple message is harmless, the act of targeting based on time zone has caused this package to be flagged by various scanners. Unfortunately the package author does not seem to want to remove it.
+
+So we have been working on a way to get around this. [@confused-techie] has managed to [pin this to an earlier version](https://github.com/pulsar-edit/pulsar/pull/608) using our own fork of this package.
+
+In our experiments, binaries built in this manner are no longer being flagged by any notable antivirus program. Hopefully this gives our users peace of mind.
+
+## Pulsar available on deb-get
+
+<img src="./assets/deb-get-logo.png" height=200>[^2]
+
+We want to put Pulsar on all major package managers, but it can be trickier than it looks. Various package managers have special requirements and restrictions, so it behoves us to move slowly and make sure we understand those constraints.
```

Comment:
```suggestion
We want to put Pulsar on all major package managers, but it can be trickier than it looks. Various package managers have special requirements and restrictions, so it behooves us to move slowly and make sure we understand those constraints.
```

Reply:
This isn't a typo?

Reply:
Apparently:

>“Behoves” and “behooves” are both correct spellings of the word “behoove”. “Behove” is the British English spelling.

Reply:
Well that would explain it lol

Reply:
Yeah, so I'd say since you normally try to go for an EN-GB style here, lets leave it at `behooves`

Reply:
That's a new regional variant for me. I now have no opinion on the outcome here. :)